packages/color-scale-block/src/types.ts

```diff
@@ -0,0 +1,113 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { MouseEvent, ReactChild } from 'react';
+import { ColorFormat, DraggableItem, DropZonePosition, Palette } from '@frontify/fondue';
+import { FrontifyColor, FrontifyColorPalette } from '@frontify/app-bridge';
+
+export type Settings = {
+    customHeight: boolean;
+    heightSlider: string;
+    heightInput: string;
+    colorInput: ColorProps[];
+};
+
+export type ColorPickerFlyoutColor = {
+    id?: number | undefined;
+    alpha?: number | undefined;
+    red: number | undefined;
+    green: number | undefined;
+    blue: number | undefined;
+    name: string | undefined;
+};
+
+export type ColorProps = {
```

Comment:
This is the exact same type as the one from AppBridge

Reply:
`width` does not exist on FrontifyColor